gltf/gltfpack.cpp

```diff
@@ -183,6 +189,70 @@ static void printImageStats(const std::vector<BufferView>& views, TextureKind ki
 		printf("stats: image %s: %d bytes in %d images\n", name, int(bytes), int(count));
 }
 
+static bool printMergeMetadata(const char* path, const std::vector<Mesh>& meshes) {
```

Comment:
nit: I think the default style of these cpp files is to follow _Allman_ style for curly brackets. Although I am team K&R, might want to keep things consistent. This is super minor so feel free to disregard this and other instances of this.

Reply:
Thx, I did not even realize!
Addressed all comments in 538a0032138e89aa35da4f4d419b14fabfd8b8ac